.github/workflows/tests.yml

```diff
@@ -19,9 +19,9 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, "3.10"]
```

Comment:
I guess we should quote all the versions (that seems to be the "recommendation" from action authors: https://github.com/actions/setup-python#usage).